From c0b3a7fbf668320f7710c85c8e9799ad3722300c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Wed, 2 Oct 2013 10:08:33 +0200
Subject: [PATCH] Work-around: link libsystemd-label to libsystemd-login.

This has been fixed upstream by moving the code around but adding
the linking gives much less patching.
---
 Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile.am b/Makefile.am
index 8d9c587..dc85ac3 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -3918,6 +3918,7 @@ libsystemd_login_la_LDFLAGS = \
 libsystemd_login_la_LIBADD = \
 	libsystemd-shared.la \
 	libsystemd-daemon-internal.la \
+	libsystemd-label.la \
 	$(RT_LIBS)
 
 libsystemd_login_internal_la_SOURCES = \
-- 
1.8.3.2

